README.md

```diff
@@ -11,7 +11,7 @@ Usage
 -----
 
     make
-    echo "0 1\n0 2\n1 0\n1 3\n2 0\n3 0\n3 1\n3 2" | java de.normalisiert.utils.graphs.TestCycles 4
+    echo -e "0 1\n0 2\n1 0\n1 3\n2 0\n3 0\n3 1\n3 2" | java de.normalisiert.utils.graphs.TestCycles 4
```

Comment:
`echo -e` is not portable. The right fix would be to use printf.

Reply:
I am not a shell expert, but from what I read you are correct. Printf is the more portable option.